Module4_Labs/Lab7_Zoologist/Cards/41.md

```diff
@@ -0,0 +1,26 @@
+Remember that Big O notation describes the behavior of functions as input grows, in this case as the number of elements needed to sort increases.
+
+**Bubble Sort**
+
+For bubble sort, there is a different time complexity for best case and worst case. This is important to note because that means that after a certain number of iterations, or elements to sort, this method becomes very inefficient. In the best case, the time it takes to sort is O(n), but this is only in the case that it sorts an already sorted list, because no sorting needs to happen.
```

Comment:
can you possibly make references to the code snippets? Like "notice how there is a nested for loop" kind of thing?